### ISSUE TITLE
[BUG] Cannot Find module nx/src/utils/app-root with nx 14.2

### ISSUE THREAD

**Issue by EelcoLos** [nx-dotnet/nx-dotnet#451]
**Describe the bug**
NX 14.2.0 / 14.2.1 requests a nx-dotnet/core:restore with npm install. This results in an error and cannot be resolved

**To Reproduce**

1. Install latest nx (time of writing 08-06-2022) with `nx migrate latest`
2. run `nx migrate --run-migrations`

**Expected behavior**
A working nx-dotnet/core restore generator command with nx

**Screenshots**
the error shown is 

```
Cannot find module 'nx/src/utils/app-root'
Require stack:
- ~\node_modules\@nx-dotnet\utils\src\lib\utility-functions\glob.js
- ~\node_modules\@nx-dotnet\utils\src\lib\utility-functions\index.js
- ~\node_modules\@nx-dotnet\utils\src\lib\index.js
- ~\node_modules\@nx-dotnet\utils\src\index.js
- ~\node_modules\@nx-dotnet\dotnet\src\lib\core\dotnet.client.js
- ~\node_modules\@nx-dotnet\dotnet\src\lib\core\index.js
- ~\node_modules\@nx-dotnet\dotnet\src\lib\index.js
- ~\node_modules\@nx-dotnet\dotnet\src\index.js
- ~\node_modules\@nx-dotnet\core\src\generators\restore\generator.js
- ~\node_modules\nx\src\config\workspaces.js
- ~\node_modules\nx\src\command-line\generate.js
- ~\node_modules\nx\src\command-line\nx-commands.js
- ~\node_modules\nx\bin\init-local.js
- ~\node_modules\nx\bin\nx.js
```

**Environment:**

- OS: Windows 11 21H2
- Browser NA
- Version 1.11.0
- Affected Packages: 
  -  "@angular-devkit/architect": "0.1400.0",
  -  "@angular-devkit/build-angular": "14.0.0",
  -  "@angular-devkit/core": "14.0.0",
  -  "@angular/compiler-cli": "14.0.0",
  -  "@angular/language-service": "14.0.0",
  -  "@nrwl/angular": "14.2.1",
  -  "@nrwl/cli": "^14.1.9",
  -  "@nrwl/cypress": "14.2.1",
  -  "@nrwl/eslint-plugin-nx": "14.2.1",
  -  "@nrwl/jest": "14.2.1",
  -  "@nrwl/linter": "14.2.1",
  -  "@nrwl/nx-cloud": "14.0.8",
  -  "@nrwl/workspace": "14.2.1",
  -  "@nx-dotnet/core": "^1.11.0",
  -  "@typescript-eslint/eslint-plugin": "5.24.0",
  -  "@typescript-eslint/parser": "5.24.0",
  -  "eslint": "8.15.0",

**Additional context**
updating cli didn't improve the commands

**Comment by EelcoLos** [comment]
I checked the source file of approot on the nx folder, which states the following:

```typescript
/**
 * The root of the workspace.
 *
 * @deprecated use workspaceRoot instead
 */
export declare const appRootPath: string;
/**
 * The root of the workspace
 */
export declare const workspaceRoot: string;
export declare function fileExists(filePath: string): boolean;
```

**Comment by AgentEnder** [comment]
Yeah, 14.2 moved it. I'll get a release out soon to support it.

IIRC I can import this from devkit now so it shouldn't break compatibility with previous Nx versions. I'll try to track down when the re-export in devkit was added so we can establish a minimum compatibility version

**Comment by AgentEnder** [comment]
This should be fixed as of v1.12.0
